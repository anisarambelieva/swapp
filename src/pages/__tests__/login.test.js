import { render } from '@testing-library/react';

import Login from '../login.js';

describe('<Login/>', () => {
  it('should render Login page with correct styles', () => {
    const { container } = render(<Login />);

    expect(container).toMatchSnapshot();
  });
});
