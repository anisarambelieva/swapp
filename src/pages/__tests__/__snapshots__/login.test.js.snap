// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> should render Login page with correct styles 1`] = `
.c6 {
  font-weight: bold;
  color: #ffe300;
  background-color: black;
  border: 0;
}

.c6:hover {
  color: black;
  background-color: #ffe300;
}

.c2 {
  font-family: starWarsFont;
  font-size: 90px;
  color: #ffe300;
  margin: 0;
  padding-top: 0.2em;
}

.c0 {
  background-color: black;
  padding-bottom: 10px;
  width: 600px;
}

.c3 {
  background-color: white;
  padding: 30px 0;
  margin: 10px;
  border-radius: 10px;
}

.c4 {
  margin-bottom: 20px;
}

.c4:last-of-type {
  margin-bottom: 0;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: right;
  -webkit-justify-content: right;
  -ms-flex-pack: right;
  justify-content: right;
}

<div>
  <div
    class="c0 container"
  >
    <div
      class="row"
    >
      <div
        class="c1 col-md-8 offset-md-2"
      >
        <h1
          class="c2"
        >
          SWAPP
        </h1>
      </div>
    </div>
    <div
      class="c3 row"
    >
      <div
        class="col"
      >
        <div
          class="c4 row"
        >
          <div
            class="c1 col-md-8 offset-md-2"
          >
            <input
              class="form-control"
              placeholder="username"
            />
          </div>
        </div>
        <div
          class="c4 row"
        >
          <div
            class="c1 col-md-8 offset-md-2"
          >
            <input
              class="form-control"
              placeholder="password"
            />
          </div>
        </div>
        <div
          class="c4 row"
        >
          <div
            class="c5 col-md-2 offset-md-8"
          >
            <button
              class="c6 btn btn-primary"
              type="button"
            >
              Login
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
